### PR TITLE
Initial implementation of converting flow rate to volume for metrics

### PR DESCRIPTION
**Pull request by dvdhoz**
Adding methods for computing a volume from a rate for MaxAccumDuration, CumulativeComputable, and Total

### CODE REVIEW

**Comment by dvdhoz** [comment]
I had to update a few of the tests so they are comparing volumes instead of flow rates

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MedianComputable.java [hunk 1]
```diff
@@ -13,6 +17,20 @@ public float compute(float[] values) {
         return (values[(n)/2-1] + values[n /2]) / 2;
     }
 
+    private void getInputUnits() {
+        if(config == null || config.getUnits().isEmpty()) {
+            outputUnit = DEFAULT_INPUT_UNITS;
```
Comment (Brennan1994) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MedianComputable.java:22]: This is weird to me. Why does getInputUnits set output unit and not return anything? potential copy paste error going on her?

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MinComputable.java [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
-            }
-        }
-        return returnvalue;*/
         java.util.Arrays.sort(values);
         return values[0];
     }
 
+    private void getInputUnits() {
+        if(config == null || config.getUnits().isEmpty()) {
+            outputUnit = DEFAULT_INPUT_UNITS;
```
Comment (Brennan1994) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MinComputable.java:15]: same comment. This seems weird

File: FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/PercentilesComputable.java [hunk 3]
```diff
@@ -2,39 +2,56 @@
...
     @Override
     public float compute(float[] values) {
         Arrays.sort(values);
-        return computePercentile(values, _percentiles[0]);
+        return computePercentile(values, selectedPercentiles[0]);
+    }
+
+    private void getInputUnits() {
+        if(config == null || config.getUnits().isEmpty()) {
+            outputUnits = DEFAULT_INPUT_UNITS;
```
Comment (Brennan1994) [inline FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/PercentilesComputable.java:34]: Ok, this is my primary gripe. Lets talk about this

**Review by Brennan1994 — CHANGES_REQUESTED** [review]

**Review by Brennan1994 — APPROVED** [review]

**Pull request merged**